### PR TITLE
Adding pip-0003, removing sequence from transaction

### PR DESCRIPTION
**Pull request by b00f**
This PR adds a new PIP to remove sequence from the transactions.

### CODE REVIEW

**Comment by b00f** [comment]
This is a log for duplicated stamps in Testnet-1:

```
duplicated stamp (<STAMP>) FROM -> TO [DISTANCE]
duplicated stamp (f151c5bd) 24006 -> 59901 [35895]
duplicated stamp (e348d488) 40067 -> 143700 [103633]
duplicated stamp (8188c500) 143009 -> 187871 [44862]
duplicated stamp (b3f24d68) 10316 -> 195908 [185592]
duplicated stamp (270f9e03) 206395 -> 210177 [3782]
duplicated stamp (e98a7a29) 66848 -> 247840 [180992]
duplicated stamp (cab849df) 118492 -> 259074 [140582]
duplicated stamp (043b1e7e) 225999 -> 263131 [37132]
duplicated stamp (439203df) 42617 -> 284782 [242165]
duplicated stamp (bc441820) 53914 -> 300028 [246114]
duplicated stamp (2b2c7238) 303620 -> 341469 [37849]
duplicated stamp (206999f0) 10011 -> 348936 [338925]
duplicated stamp (f06f41aa) 282906 -> 362224 [79318]
duplicated stamp (34a0b4e2) 345718 -> 364428 [18710]
duplicated stamp (f0270f8c) 118799 -> 412880 [294081]
duplicated stamp (9be51d68) 389930 -> 426388 [36458]
duplicated stamp (121a02b8) 411457 -> 440726 [29269]
duplicated stamp (81c8a36f) 124478 -> 444033 [319555]
duplicated stamp (7fef1981) 88414 -> 461367 [372953]
duplicated stamp (ec700a5f) 120583 -> 461524 [340941]
duplicated stamp (dff808b9) 261943 -> 463950 [202007]
duplicated stamp (833acec3) 77133 -> 474868 [397735]
duplicated stamp (e50f564b) 302041 -> 512110 [210069]
duplicated stamp (ef7df259) 221515 -> 516013 [294498]
duplicated stamp (7764c2d8) 422719 -> 550713 [127994]
duplicated stamp (9f6ff4e8) 392438 -> 551123 [158685]
duplicated stamp (b111001e) 170810 -> 555701 [384891]
duplicated stamp (99022ae9) 444008 -> 572750 [128742]
duplicated stamp (6af74dfe) 373966 -> 578000 [204034]
duplicated stamp (cc940b73) 272063 -> 578090 [306027]
duplicated stamp (e750beba) 525013 -> 591049 [66036]
duplicated stamp (50a89632) 36990 -> 592138 [555148]
duplicated stamp (23b93cc2) 123860 -> 592903 [469043]
duplicated stamp (81d696e7) 44254 -> 624712 [580458]
```

File: PIPs/pip-0003.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+created: 2023-08-23
+---
+
+## Abstract
+
+This document propose to remove the `Sequence` field from the transactions.
+
+## Motivation
+
+The `Sequence` in Pactus is similar to the `nonce` in Ethereum.
```
Comment (Ja7ad) [inline PIPs/pip-0003.md:15]: Please add transaction data in code here for clean Motivation details.

```go
type Tx struct {
	memorizedID  *ID
	basicChecked bool

	data txData
}

type txData struct {
	// Version format
	//  7             0
	// +-+-+-+-+-+-+-+-+
	// |L|R|R|R|VERSION|
	// +-+-+-+-+-+-+-+-+
	// L: Lock Time transaction
	// R: Reserved bit
	//
	Version   uint8
	Stamp     hash.Stamp
	LockTime  uint32
	Sequence  int32
	Fee       int64
	Payload   payload.Payload
	Memo      string
	PublicKey crypto.PublicKey
	Signature crypto.Signature
}
```

**Review by Ja7ad — CHANGES_REQUESTED** [review]

File: PIPs/pip-0003.md (hunk 1 above)
Comment (b00f) [reply PIPs/pip-0003.md:15]: We can add link to the website: https://pactus.org/learn/transaction/format/

Feel free to edit it

**Comment by b00f** [comment]
In each block we should make sure the transaction hashes are not duplicated.

File: PIPs/pip-0003.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  The current placeholder is acceptable for a draft.
+
+  TODO: Remove this comment before submitting
+-->
+
+Needs discussion.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](../LICENSE.md).
```
Comment (Ja7ad) [inline PIPs/pip-0003.md:53]: Base on pipeline error file `LICENSE.md` not found.

https://github.com/pactus-project/PIPs/actions/runs/6069455601/job/16463863847?pr=5#step:5:17

**Review by Ja7ad — CHANGES_REQUESTED** [review]

**Review by allahbakhsh — APPROVED** [review]

**Review by Ja7ad — APPROVED** [review]

**Pull request merged**